lib/topological_inventory/api/logger.rb

```diff
@@ -0,0 +1,37 @@
+require "manageiq/loggers"
+
+module TopologicalInventory
+  module Api
+    class << self
+      attr_writer :logger
+    end
+
+    def self.logger
+      @logger ||= ManageIQ::Loggers::CloudWatch.new
```

Comment:
I don't think we should open another connection to CloudWatch.  `Rails.logger` should already be a CloudWatch logger.

Reply:
yes you are right. I took a look how `CloudWatch ` instance is created.
Rails's logger used👍